<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGUI</class>
 <widget class="QMainWindow" name="MainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>814</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>FIMTrack v2.1</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeWidget" name="treeView">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>620</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>310</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Job</string>
           </property>
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Path</string>
           </property>
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoad">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Load Job</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelete">
          <property name="text">
           <string>Delete Job</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReset">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Reset All Jobs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelCameraParameter">
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="TrackerGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Helvetica</family>
             </font>
            </property>
            <property name="text">
             <string>Gray Threshold</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_graythresh">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Helvetica</family>
             </font>
            </property>
            <property name="text">
             <string>Min Larvae Area</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_minSizeThresh">
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>150</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Helvetica</family>
             </font>
            </property>
            <property name="text">
             <string>Max Larvae Area</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_maxSizeThresh">
            <property name="maximum">
             <number>9999999</number>
            </property>
            <property name="value">
             <number>450</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxShowTrackingProgress">
            <property name="font">
             <font>
              <family>Helvetica</family>
             </font>
            </property>
            <property name="text">
             <string>Show Tracking Progress</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="btnPreview">
            <property name="font">
             <font>
              <family>Helvetica</family>
             </font>
            </property>
            <property name="text">
             <string>Preview</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnTrack">
            <property name="font">
             <font>
              <family>Helvetica</family>
             </font>
            </property>
            <property name="text">
             <string>Track</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionLoad_Camera_Setting"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionOpen_LOG_Window"/>
    <addaction name="separator"/>
    <addaction name="actionResults_Viewer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>../tango-icon-theme-0.8.90/32x32/actions/document-new.png</normaloff>../tango-icon-theme-0.8.90/32x32/actions/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../tango-icon-theme-0.8.90/32x32/actions/document-open.png</normaloff>../tango-icon-theme-0.8.90/32x32/actions/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>../tango-icon-theme-0.8.90/32x32/actions/document-save.png</normaloff>../tango-icon-theme-0.8.90/32x32/actions/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <italic>false</italic>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normaloff>../tango-icon-theme-0.8.90/32x32/actions/document-save-as.png</normaloff>../tango-icon-theme-0.8.90/32x32/actions/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>../tango-icon-theme-0.8.90/32x32/actions/system-log-out.png</normaloff>../tango-icon-theme-0.8.90/32x32/actions/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLoad_Camera_Setting">
   <property name="icon">
    <iconset>
     <normaloff>../tango-icon-theme-0.8.90/32x32/devices/camera-photo.png</normaloff>../tango-icon-theme-0.8.90/32x32/devices/camera-photo.png</iconset>
   </property>
   <property name="text">
    <string>Load Camera Setting</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+C, Ctrl+L</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset>
     <normaloff>../tango-icon-theme-0.8.90/32x32/categories/preferences-system.png</normaloff>../tango-icon-theme-0.8.90/32x32/categories/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionOpen_LOG_Window">
   <property name="icon">
    <iconset>
     <normaloff>../tango-icon-theme-0.8.90/32x32/actions/edit-paste.png</normaloff>../tango-icon-theme-0.8.90/32x32/actions/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>Open LOG-Window</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+L, Ctrl+O</string>
   </property>
  </action>
  <action name="actionResults_Viewer">
   <property name="icon">
    <iconset>
     <normaloff>../tango-icon-theme-0.8.90/32x32/apps/accessories-text-editor.png</normaloff>../tango-icon-theme-0.8.90/32x32/apps/accessories-text-editor.png</iconset>
   </property>
   <property name="text">
    <string>Results Viewer...</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TrackerGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>GUI/TrackerGraphicsView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
